audio/src/main/java/edu/artic/audio/AudioLookupViewModel.kt

```diff
@@ -106,20 +111,29 @@ class AudioLookupViewModel @Inject constructor(
 
         val objectFoundObservable = lookupRequests
                 .observeOn(Schedulers.io())
-                .map {
-                    Optional(objectLookupDao.getObjectBySelectorNumber(it))
+                .map { objectSelectorNumber ->
+
+                    val likeCondition = """
+                        %"object_selector_number":"$objectSelectorNumber"%
+                    """.trimIndent()
```

Comment:
I'm not entirely happy with having this condition here. Wouldn't it make more sense for that to be a constant in the DAO interface or the model class?

Reply:
Good idea @Cliabhach. Moved it to DAO interface.